### ISSUE TITLE
Larger directory preview on `tab tab`

### ISSUE THREAD

**Issue by JustChris** [warpdotdev/Warp#2659]
### Describe the solution you'd like?

```text
I would like to have a full or even larger directory preview when I `tab tab` into a directory.

Currently I cannot get a proper preview of the directory content before I execute the actual command. 
So I am forced to list the entire directory first, which is quite annoying.

The screenshot below will make clear what I mean.
```

### Additional context

This is how warp does it currently:
![Screenshot 2023-02-16 at 08 40 10](https://user-images.githubusercontent.com/68645707/219300553-097c7512-30d1-4699-b606-15c68304a058.png)

This is how other terminals do it and I would love to have in warp too!
![Screenshot 2023-02-16 at 08 47 33](https://user-images.githubusercontent.com/68645707/219301051-fe27a7e4-99e6-4161-ade2-411b06c1485c.png)

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

None

**Comment by dannyneira** [comment]
Thanks for submitting @
We have an issue tracking this here: https://github.com/warpdotdev/warp/issues/2583

TLDR, We're working on a way to resize history or suggestion tabs. So if you're interested please :+1: and subscribe to that request as it helps us gauge interest. We'll post any updates there.

Closing as duplicate.
